documentation/modules/exploit/windows/misc/ivanti_avalanche_mdm_bof.md

```diff
@@ -0,0 +1,60 @@
+## Vulnerable Application
+
+Ivanti Avalanche powered by Wavelink is a mobile device management system. Network security
+features allow you to manage wireless settings (including encryption and authentication),
+and apply those settings on a schedule throughout the network. Software distribution features
+allow you to schedule software distribution to specific devices from the Avalanche Console.
+Avalanche also provides tools for managing alerts and reports.
+
+This module exploits a buffer overflow condition in Ivanti Avalanche MDM versions before `v6.4.1`.
+An attacker can send a specially crafted message to the Wavelink Avalanche Manager,
+which could result in arbitrary code execution with the `NT/AUTHORITY SYSTEM` permissions.
+This vulnerability occurs during the processing of 3/5/8/100/101/102 item data types.
+The program tries to copy the item data using `qmemcopy` to a fixed size data buffer on stack.
+Upon successful exploitation the attacker gains full access to the target system.
+
+The original analysis and the vulnerability discovery is done by the Tenable.
+Check [here](https://www.tenable.com/security/research/tra-2023-27) for public advisory.
+
+## Testing
+For installing the vulnerable version follow the steps below,
+1. To obtain the vulnerable versions of the MDM setup, first create a customer account at
+[Ivanti](https://success.ivanti.com/customers/Community_RegStep1_Page?lp=register) (trial license is sufficient)
+2. Navigate [here](https://www.wavelink.com/Download-Avalanche_Mobile-Device-Management-Software/)
+and download any version **below** `v6.4.1`
+3. Follow the installation steps.
+
+After these steps, the MDM service should be accessible on port 1777.
```

Comment:
I would maybe add a note here about potentially needing to restart the `Wavelink Avalanche Manager` if they are having issues.